src/styles/ftva/_divider-way-finder.scss

```diff
@@ -0,0 +1,9 @@
+.ftva.divider-way-finder {
+  &.color-default {
+    --color-border: var(--color-default-cyan-03);
```

Comment:
is this default color ever used in ftva? I don't see it in the mockups. Can we make the default the accent-blue so we don't have to pass the 'ftva' color prop everytime we use the divider on ftva?